public/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "theme": {
+    "@primary-color": "#1890ff",
```

Comment:
@utchoang are these the complete list of variables, this has more variables - https://www.antdv.com/docs/vue/customize-theme/#Ant-Design-Vue-Less-variables ? Thanks (excluding fonts is okay I think). cc @svenvogel ?

Reply:
@rhtyd I checked that there are some attributes like @ link-color that don't seem to be used in the project so I deleted them. So you will notice that there are some incomplete attributes.